src/Model/Donor.php

```diff
@@ -50,10 +52,88 @@ class Donor
      */
     private $mandateSource;
 
-    public function __construct(DonorState $state, string $mandateSource)
-    {
+    /**
+     * @var string
+     */
+    private $payerNumber;
+
+    /**
+     * @var \byrokrat\banking\AccountNumber
+     */
+    private $account;
+
+    /**
+     * @var \byrokrat\id\Id
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $comment;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var \byrokrat\giroapp\Model\PostalAddress
```

Comment:
Oh, good. I sometimes get confused over the namespaces, and over-specify just to be on the safe side...